backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/HybridBackupCommand.java

```diff
@@ -237,6 +237,31 @@ private void auditLog(AuditLogType type) {
         auditLogDirector.log(this, type);
     }
 
+    @Override
+    public AuditLogType getAuditLogTypeValue() {
+        VmBackup vmBackup = getParameters().getVmBackup();
+        addCustomValue("VmName", getVm().getName());
+        addCustomValue("backupId", vmBackup.getId().toString());
```

Comment:
the convention is to use capital letters at the beginning, no?

Reply:
yeah, but it's copy-paste from StartVmCommand